Source/FieldSolver/SpectralSolver/SpectralKSpace.cpp

```diff
@@ -178,48 +182,65 @@ SpectralKSpace::getModifiedKComponent( const DistributionMapping& dm,
     // Initialize an empty ManagedVector in each box
     KVectorComponent modified_k_comp(spectralspace_ba, dm);
 
-    // Compute real-space stencil coefficients
-    Vector<Real> stencil_coef = getFonbergStencilCoefficients(n_order, nodal);
+    if(n_order == -1){ // Infinite-order case
+        for ( MFIter mfi(spectralspace_ba, dm); mfi.isValid(); ++mfi ){
+            Real delta_x = dx[i_dim];
```

Comment:
I think it should be possible to remove the declaration of `delta_x` from this block, it does not seem to be used:
```suggestion
```